tests/transport_test.go

```diff
@@ -71,25 +79,41 @@ var _ = Describe("Transport Tests", func() {
 			pvcAnn[controller.AnnSecret] = sec.Name
 		}
 
-		By(fmt.Sprintf("Creating PVC with endpoint annotation %q", ep))
+		By(fmt.Sprintf("Creating PVC with endpoint annotation %q", pvcAnn[controller.AnnEndpoint]))
 		pvc, err := utils.CreatePVCFromDefinition(c, ns, utils.NewPVCDefinition("transport-e2e", "20M", pvcAnn, nil))
 		Expect(err).NotTo(HaveOccurred(), "Error creating PVC")
 
-		err = utils.WaitForPersistentVolumeClaimPhase(c, ns, v1.ClaimBound, pvc.Name)
-		Expect(err).NotTo(HaveOccurred(), "Error waiting for claim phase Bound")
+		importer, err := utils.FindPodByPrefix(c, ns, common.IMPORTER_PODNAME, common.CDI_LABEL_SELECTOR)
+		Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Unable to get importer pod %q", ns + "/" + common.IMPORTER_PODNAME))
+
+		Expect(utils.WaitTimeoutForPodStatus(c, importer.Name, importer.Namespace, v1.PodSucceeded, utils.PodWaitForTime)).To(Succeed())
 
 		By("Verifying PVC is not empty")
-		Expect(framework.VerifyPVCIsEmpty(f, pvc)).To(BeFalse())
+		Expect(framework.VerifyPVCIsEmpty(f, pvc)).To(BeFalse(), fmt.Sprintf("Found 0 imported files on PVC %q", pvc.Namespace + "/" + pvc.Name))
+
+		By(fmt.Sprintf("Verifying imported file size matches %q size of %d bytes", targetFile, targetSize))
+		pod, err := utils.CreateExecutorPodWithPVC(c, sizeCheckPod, ns, pvc)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(utils.WaitTimeoutForPodReady(c, sizeCheckPod, ns, 20*time.Second)).To(Succeed())
+
+		stdout := f.ExecShellInPod(pod.Name, ns, "wc -c < /pvc/disk.img")
```

Comment:
should this be a const or a var? I know we only import to _disk.img_ at the moment, but I suspect the user will be able to name the target in the fairly near future...

Reply:
Given that that functionality doesn't exist yet, I would expect tests to be updated when it was added.